service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java

```diff
@@ -124,6 +139,9 @@ void setup() {
 
         given(localAuthorityUserLookupConfiguration.getUserIds(LA_IN_PRD_CODE))
             .willThrow(new UnknownLocalAuthorityCodeException(LA_IN_PRD_CODE));
+
+        given(caseDataAccessApi.addCaseUserRoles(anyString(), anyString(), any(AddCaseAssignedUserRolesRequest.class)))
```

Comment:
probably should not be here, only 2 out of 11 tests needs this
you can have @Nested with @BeforeEach where you can do this initialization

Reply:
as discussed , we tried it but nested has issues, hence reverting that change and initialising that instance in individual tests.